orttraining/orttraining/training_ops/cuda/optimizer/gradient_control_impl.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+namespace onnxruntime {
+namespace cuda {
+// Implementation can be found in cuda file, optimizers_impl.cu
```

Comment:
>optimizers_impl.cu [](http://example.com/codeflow?start=45&length=18)

is it in gradient_control_impl.cu?

Reply:
I just moved this comment. I will fix it.